libs/ui/src/ui_strings/ui_strings_context.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { LanguageCode } from '@votingworks/types';
+import { waitFor } from '../../test/react_testing_library';
+import { newTestContext } from '../../test/ui_strings/test_utils';
+
+test('includes both language and audio contexts', async () => {
+  const { getAudioContext, getLanguageContext, mockBackendApi, render } =
+    newTestContext();
+  mockBackendApi.getAvailableLanguages.mockResolvedValue([
+    LanguageCode.ENGLISH,
+  ]);
+  mockBackendApi.getUiStrings.mockResolvedValue(null);
+
+  render(<div>foo</div>);
+
+  await waitFor(() => expect(getLanguageContext()).toBeDefined());
+  expect(getAudioContext()).toBeDefined();
+});
+
+test('skips audio context when `noAudio` is true', () => {
+  const { getAudioContext, getLanguageContext, mockBackendApi, render } =
+    newTestContext({ noAudio: true });
+  mockBackendApi.getAvailableLanguages.mockResolvedValue([
+    LanguageCode.ENGLISH,
+  ]);
+  mockBackendApi.getUiStrings.mockResolvedValue(null);
+
+  render(<div>foo</div>);
+
+  expect(getLanguageContext).toBeDefined();
```

Comment:
I think you probably meant to add parentheses after `getLanguageContext` here (and wrap in a `waitFor` as needed)

Reply:
oh, thanks for catching 😅